backend/estimate-service/src/main/java/org/egov/works/repository/rowmapper/EstimateRowMapper.java

```diff
@@ -66,7 +66,7 @@ public List<Estimate> extractData(ResultSet rs) throws SQLException, DataAccessE
 
             Estimate estimate = Estimate.builder().estimateNumber(estimateNumber).id(UUID.nameUUIDFromBytes(id.getBytes(StandardCharsets.UTF_8)))
                     .estimateStatus(estimateStatus).status(Estimate.StatusEnum.fromValue(status)).adminSanctionNumber(adminSanctionNumber)
-                    .totalAmount(totalAmount).additionalDetails(additionalDetails).beneficiaryType(beneficiaryType).budgetHead(budgetHead)
+                    .totalAmount(totalAmount).additionalDetails(additionalDetails.asText()).beneficiaryType(beneficiaryType).budgetHead(budgetHead)
```

Comment:
Why asText()? We need to store this in db as json.

Reply:
it's for reading from DB. and while reading , I am sending as String object. Hence we need asText().

Reply:
Has it been implemented in the same way in other digit services?

Reply:
no..I have modified as digit service has done.

Reply:
Another question, why are you storing this json object as string in db?

Reply:
This class is for extracting the result that we are querying from DB. And I am not storing this as String in DB . I am just pushing the same object (in case of create/ update) to the kafka topic for persister.